src/features/tags/selectors.ts

```diff
@@ -0,0 +1,79 @@
+import { NAME } from "./constants";
+import { Tag } from "./types";
+
+// does created_at get updated whenever a row is updated in mysql?
+export const getMostRecentTags = (
+  state: any,
+  list: string
+): Tag[] | undefined => {
+  // give time for tags to load
+  if (state.tagsByList[`${list}`] && state.tagsByList[`${list}`].items) {
+    let tagsArray = state.tagsByList[`${list}`].items.map(
+      (id: number) => state[NAME][id]
+    );
+
+    let tags = tagsArray
+      .sort((a: Tag, b: Tag) => {
+        if (a.date > b.date) return 1;
+        else return -1;
+      })
+      .map((tag: Tag) => state[NAME][tag.id].id)
```

Comment:
Right here is here it maps to just the tag id instead of the whole tag object. (No action needed here)

Reply:
Ahh I missed this. Thanks for pointing it out.